### PR TITLE
Add some clarifying info about the main OpenCue components.

### PR DESCRIPTION
**Pull request by bcipriano**
Fixes https://github.com/imageworks/OpenCue/issues/220

Add info on database/Cuebot/client-side tools to clarify where they're supposed to be installed.

### CODE REVIEW

File: content/en/docs/Getting started/deploying-cuebot.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ management tasks, including:
 A typical OpenCue deployment runs a single instance of Cuebot, which is shared
 by all users.
 
+Cuebot functions best when it has a low-latency connection to the
```
Comment (sharifsalah) [inline content/en/docs/Getting started/deploying-cuebot.md]: Wondering if this fits more naturally under 'System requirements' directly below?

Also suggest rewording slightly:

Cuebot functions best when it has a low-latency connection to the
[OpenCue database](/docs/getting-started/setting-up-the-database). We recommend that
you run Cuebot, either on the same machine or within the same local network as the database.

File: content/en/docs/Getting started/installing-pycue-and-pyoutline.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ specification XML, allowing you to construct complex jobs with Python code
 instead of working directly with XML. PyOutline is used by CueSubmit to
 construct its job submissions.
 
+PyCue and PyOutline will need to be available on all systems which want to make use of
```
Comment (sharifsalah) [inline content/en/docs/Getting started/installing-pycue-and-pyoutline.md]: I suggest tweaking a bit:

Install the PyCue and PyOutline libraries on the systems of all users
that want to use them. Make sure you also install PyCue and PyOutline on all
systems that run tools such as CueGUI and CueSubmit, which depend on the
libraries.

File: content/en/docs/Getting started/setting-up-the-database.md [hunk 3]
```diff
@@ -10,6 +10,16 @@ description: >
 This page describes how to install and set up a
 [PostgreSQL](https://www.postgresql.org/) database for use with OpenCue.
 
+OpenCue supports a single database per deployment, which serves as the single source of truth
+within your deployment.
+
+The [Cuebot(s)](/docs/getting-started/deploying-cuebot) of your deployment are the only
```
Comment (sharifsalah) [inline content/en/docs/Getting started/setting-up-the-database.md]: Looks good! 

Perhaps add a heading for 'System requirements' to cover the new content you're adding to this page?(You shouldn't need to move anything)

Also, suggest a couple of small edits:

[Cuebot](/docs/getting-started/deploying-cuebot) servers are the only
component that access the database directly. All other components, such as
[PyCue](/docs/getting-started/installing-pycue-and-pyoutline), interact with the database
indirectly via the Cuebot's gRPC API. For this reason, make sure Cuebot has a
low-latency connection to the database, either by running both on the same machine or on the
same local network.

**Comment by bcipriano** [comment]
Comments addressed. PTAL

File: content/en/docs/Getting started/deploying-cuebot.md [hunk 4]
```diff
@@ -20,9 +20,13 @@ management tasks, including:
 A typical OpenCue deployment runs a single instance of Cuebot, which is shared
 by all users.
 
+
 ## System requirements
 
-A Cuebot server requires a mimimum of 6GB of RAM.
+*   A Cuebot server requires a mimimum of 6GB of RAM.
```
Comment (sharifsalah) [inline content/en/docs/Getting started/deploying-cuebot.md]: As the requirements are now a list, it would be good to add a simple introductory sentence, something like:

To plan your installation of Cuebot, consider the following requirements:

File: content/en/docs/Getting started/setting-up-the-database.md [hunk 5]
```diff
@@ -10,6 +10,18 @@ description: >
 This page describes how to install and set up a
 [PostgreSQL](https://www.postgresql.org/) database for use with OpenCue.
 
+OpenCue supports a single database per deployment, which serves as the single source of truth
+within your deployment.
+
+## System requirements
+
+*   [Cuebot](/docs/getting-started/deploying-cuebot) servers are the only
```
Comment (sharifsalah) [inline content/en/docs/Getting started/setting-up-the-database.md]: As it's just a single item, I recommend removing the list formatting here.

**Comment by sharifsalah** [comment]
Looks great, thanks! I added a couple of minor additional points to consider.

File: content/en/docs/Getting started/deploying-cuebot.md (hunk 4 above)
Comment (bcipriano) [reply content/en/docs/Getting started/deploying-cuebot.md]: Done.

File: content/en/docs/Getting started/setting-up-the-database.md (hunk 5 above)
Comment (bcipriano) [reply content/en/docs/Getting started/setting-up-the-database.md]: Done.

**Comment by bcipriano** [comment]
Comments resolve and preview looks good. Merging.
